runtime/karura/src/lib.rs

```diff
@@ -118,7 +118,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("karura"),
 	impl_name: create_runtime_str!("karura"),
 	authoring_version: 1,
-	spec_version: 1000,
+	spec_version: 1001,
```

Comment:
no need to change this until we have mainnet

Reply:
Oh the change was for running the wasm upgrade test, and I forgot to reset it.